bindings/java/api/src/main/com/apple/foundationdb/tuple/TupleUtil.java

```diff
@@ -220,7 +214,7 @@ private static void adjustVersionPosition300(byte[] packed, int delta) {
 		buffer.putShort((short)versionPosition);
 	}
 
-	private static void adjustVersionPosition520(byte[] packed, int delta) {
+	static void adjustVersionPosition(byte[] packed, int delta) {
```

Comment:
Presumably `adjustVersionPosition300` can be deleted now? Sorry, I should have mentioned this before, but I guess I'm just noticing it now